exercises/practice/grade-school/grade-school-test.rkt

```diff
@@ -0,0 +1,195 @@
+
+      (test-case "Student is added to the roster"
+	(define school (new school%))
+        (check-true (send school add "Aimee" 2))
+	(check-equal?
+          (send school roster)
+	  '((2 "Aimee"))))
+
+      (test-case "Adding multiple students in the same grade in the roster"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-true (send school add "Paul" 2)))
+
+      (test-case "Multiple students in the same grade are added to the roster"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-true (send school add "Paul" 2))
+        (check-equal?
+	  (send school roster)
+	  '((2 "Blair" "James" "Paul"))))
+
+      (test-case "Cannot add student to same grade in the roster more than once"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-false (send school add "James" 2))
+        (check-true (send school add "Paul" 2)))
+
+      (test-case "A student can't be in two different grades"
+	(define school (new school%))
+        (check-true (send school add "Aimee" 2))
+        (check-false (send school add "Aimee" 1))
+        (check-equal?
+	 (send school roster)
+	 '((2 "Aimee"))))
+
+      (test-case "A student can only be added to the same grade in the roster once"
+	(define school (new school%))
+        (check-true (send school add "Aimee" 2))
+        (check-false (send school add "Aimee" 2))
+        (check-equal?
+	 (send school roster)
+	 '((2 "Aimee"))))
+
+      (test-case "Student not added to same grade in the roster more than once"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-false (send school add "James" 2))
+        (check-true (send school add "Paul" 2))
+        (check-equal?
+	 (send school roster)
+	 '((2 "Blair" "James" "Paul"))))
+
+      (test-case "Adding students in multiple grades"
+	(define school (new school%))
+        (check-true (send school add "Chelsea" 3))
+        (check-true (send school add "Logan" 7)))
+
+      (test-case "Students in multiple grades are added to the roster"
+	(define school (new school%))
+        (check-true (send school add "Chelsea" 3))
+        (check-true (send school add "Logan" 7))
+        (check-equal?
+	 (send school roster)
+	 '((3 "Chelsea") (7 "Logan"))))
+
+      (test-case "Cannot add same student to multiple grades in the roster"
+	(define school (new school%))
+        (check-true (send school add "Blair" 2))
+        (check-true (send school add "James" 2))
+        (check-false (send school add "James" 3))
+        (check-true (send school add "Paul" 3)))
+
+      (test-case "A student cannot be added to more than one grade in the sorted roster"
```

Comment:
This appears to be the test case 6a03b61e-1211-4783-a3cc-fc7f773fba3f which is reimplemented by the following test (c7ec1c5e-9ab7-4d3b-be5c-29f2f7a237c5). So this test needs to be removed.